tracing-appender/src/lib.rs

```diff
@@ -42,7 +107,61 @@ mod worker;
 /// Unintentional drops of `WorkerGuard` remove the guarantee that logs will be flushed
 /// during a program's termination, in a panic or otherwise.
 ///
+/// See [`WorkerGuard`][worker_guard] for examples of using the guard.
+///
+/// [worker_guard]: ./struct.WorkerGuard.html
+///
 /// # Examples
+///
+/// ``` rust
+/// # fn docs() {
+/// let (non_blocking, _guard) = tracing_appender::non_blocking(std::io::stdout());
+/// let subscriber = tracing_subscriber::fmt().with_writer(non_blocking);
+/// tracing::subscriber::with_default(subscriber.finish(), || {
+///    tracing::event!(tracing::Level::INFO, "Hello");
+/// });
+/// # }
+/// ```
+pub mod non_blocking;
+
+/// A rolling file appender.
+///
+/// Creates a log file at a specified directory and file name prefix which *may* be appended with
+/// the date and time.
```

Comment:
This doesn't really describe the behavior for actually _rolling_ the log file, which I think is more important than how dates are formatted in filenames.

Reply:
Added this information